feat: add JS only madlibs lab (#485)
freeCodeCamp/CurriculumExpansion: frontend-cert/js-projects/lab-madlibs/user-stories.md

> 1. You should define the following variables using `let`:

Reviewer: ```suggestion
1. You should declare the following variables using `let`:
```

- `adjective`
- `noun`
- `verb`
- `place`
- `adjective2`
- `noun2`

> 1. You should assign the above variables some values.

Reviewer 2: ```suggestion
1. You should assign the above variables some string values of your choice.
```

> 1. You should use the following story template to create a story:

Reviewer 3: Will the variable `let story` be given or will the learner be required to define it?

Author: The users should use the provided story template and assign it to any variable of their choice. It could be `story` or anything else.

Reviewer: @zairahira 

it will probably be easier to test to have them declare a `story` variable.

Author: added!

> `"'Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."'"`;

Reviewer 2: ```suggestion
`"Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."`;
```
Not sure if single quotes were there for a reason.

Author: These quotes shouldn't have been be there. Fixed it.

> 1. You should print the final story to the console.

Reviewer: There was an issue opened in the main repo a while back about the use of print when referring to the console.

For consistency and accuracy, I vote we use either use `log a message to the console` or `output a message to the console` like MDN says.

https://developer.mozilla.org/en-US/docs/Web/API/console/log_static

Author: done!
